core/src/main/java/io/questdb/griffin/engine/ops/InsertOperationImpl.java

```diff
@@ -110,9 +102,8 @@ public OperationFuture execute(SqlExecutionContext sqlExecutionContext) throws S
     }
 
     public void setColumnNames(ObjList<CharSequence> columnNameList) {
-        if (columnNameList.size() == 0) {
-            columnNames = EMPTY_COLUMN_LIST;
-        } else {
+        ObjList<CharSequence> columnNames;
```

Comment:
This method does not make sense now. It creates `new ObjList<>()` on the stack, adds values into it and then does not use it.
Why were we able to remove `columnNames` field, why we don't need it anymore?

Reply:
`columnNames` was unused field, intellij "removed" it badly. I'll clean up


Uploading InsertOperationImpl.java (C__Users_Vlad_dev_questdb-enterprise_questdb_core_src_main_java_io_questdb_griffin_engine_ops) 2024-11-19 11-44-54.mp4…

Reply:
https://github.com/user-attachments/assets/1d8d4e7f-a0dd-43e1-bef6-6c1ef7d67387

Reply:
cleaned up code here: https://github.com/questdb/questdb/pull/5159/commits/d81687d0b5f4a54019604f9d5ef5bcf6b51549a8